content/product/product_management/rollout_process.md

```diff
@@ -8,6 +8,37 @@ Some features have a [beta or experimental label](../beta_and_experimental_featu
 
 Product features fall into different buckets depending on how much engagement with marketing is needed. Definitions for [L1, L2, and L3 launches](../../marketing/product-marketing/marketing_launch_tiers.md) can be found in the marketing handbook.
 
+## Communications
+
+How we communicate about features is important, especially as it relates to feature roll-out.
+
+### Pre-release features
+
+We always aim to communicate clearly about pre-release features so that we do not cause confusion around customers thinking features are generally available, that they aren't going to change, or even that they couldn't be taken away if we find they aren't working.
+
+1. Prior to the official launch of the feature, we should focus in communication on the problems we are solving, rather than that feature x will definitely include scope y and/or be delivered on z date. For example, we can talk about our ultimate goal of xMM repos indexed and the reason why we are focused on it, or how we’re working on helping teams problems around a specific batch changes use case, and that we're looking for feedback. This can (and should) include some specifics, but that doesn’t need to be the emphasis because people care more about the problems we’re solving anyway.
+1. Whenever communicating publicly about pre-release features, we should use the [beta and experimental labels](../beta_and_experimental_feature_labels.md) so we don’t create confusion, and be sure we note that final capabilities of the feature might change. This includes social media, on the website, documentation, release post, changelog, handbook, in the app, or anywhere else. Documentation, as it is long-lived and not tied to a moment in time, should indicate when a feature became generally available.
+
+Public beta testing by its nature involves sharing more detail, showing and letting people use things that are still unreleased, and which can change or even be abandoned before they go live. That’s fine to do, as long as we follow the above principles.
+
+### L1 launches and milestones
+
+L1 launches ([full definition in marketing handbook](../../marketing/product-marketing/marketing_launch_tiers#l1-launch)) are special: we don't do them frequently, but when we do we engage an additional layer of collaboration around them to avoid misunderstandings around how these important features will move from development to general availability.
```

Comment:
```suggestion
L1 launches ([full definition in marketing handbook](../../marketing/product-marketing/marketing_launch_tiers.md#l1-launch)) are special: we don't do them frequently, but when we do we engage an additional layer of collaboration around them to avoid misunderstandings around how these important features will move from development to general availability.
```